python/task_manager/background_task_manager.py

```diff
@@ -1,47 +1,142 @@
 # Copyright (c) 2015 Shotgun Software Inc.
-# 
```

Comment:
So little needed to be cleaned to be PEP8 compliant that I figure we might as well clean the whole files and be done with it.